src/components/organisms/TestConfigurationForm/Git/CreationFormFields.tsx

```diff
@@ -1,22 +1,16 @@
 import {useState} from 'react';
 
-import {NamePath} from 'antd/lib/form/interface';
-
 import {Branch, Path, Repository, SecretFormItem} from '@molecules';
 
 import useValidateRepository from '@hooks/useValidateRepository';
 
 import {useValidateRepositoryMutation} from '@services/repository';
 
 import {StyledFormSpace} from '../TestConfigurationForm.styled';
+import {Props} from '../utils';
 
-type GitFormFieldsProps = {
-  executorType: string;
-  getFieldValue: (name: NamePath) => string;
-};
-
-const GitFormFields: React.FC<GitFormFieldsProps> = props => {
-  const {executorType, getFieldValue} = props;
+const GitFormFields: React.FC<Partial<Props>> = props => {
+  const {executorType, getFieldValue} = props as Pick<Props, 'executorType' | 'getFieldValue'>;
```

Comment:
`Pick` is redundant - it will pick it anyway:
```suggestion
  const {executorType, getFieldValue} = props;
```